internal/icinga2/client_api.go

```diff
@@ -230,6 +230,16 @@ func (client *Client) checkMissedChanges(ctx context.Context, objType string, ca
 			return fmt.Errorf("querying API delivered a wrong object type %q", objQueriesResult.Type)
 		}
 
+		// Downtime suppresses all kind of events on that checkable, so skip it if it's in downtime.
+		if objQueriesResult.Attrs.DowntimeDepth > 0 {
+			continue
```

Comment:
Maybe add a debug log entry similar to those in the same case within `(client *Client) listenEventStream`.